### ISSUE TITLE
pagination support

### ISSUE THREAD

**Issue by 6r6** [fireship-io/fireship.io#96]
Hi,
seems section page did not support pagination ?

**Comment by codediodeio** [comment]
Yes, we will need pagination soon as the content list grows. I'll keep this opened until it's integrated.

**Comment by 6r6** [comment]
thank you 😄, this will be a delightful feature.
